src/dto/rpc/index.ts

```diff
@@ -2,6 +2,7 @@ import { EvmBasedRpcSuite } from './EvmBasedRpcSuite'
 import { SolanaRpcSuite } from './SolanaRpcSuite'
 import { UtxoBasedRpcSuite } from './UtxoBasedRpcSuite'
 import { XrpRpcSuite } from './XrpRpcSuite'
+import {TronRpcSuite} from "./TronRpcSuite";
 
 export * from './EvmBasedRpcSuite'
 export * from './SolanaRpcSuite'
```

Comment:
add export * from './TronRpcSuite' as a default export, so we gonna have nicer imports in engine/TronRpc.ts